### ISSUE TITLE
Chat Windows

### ISSUE THREAD

**Issue by Murphy107j** [joev/SVUI-Temp#69]
When I add a new window for party chat it gets disabled when reloading ui and going through loads screen

**Comment by drveoj** [comment]
@Murphy107j Not able to reproduce this. Working perfectly for me across a number of characters and across realms. 

Can you give me some more info? e.g. what other Addons are you running which may be having a weird interaction with SVUI?

**Comment by Murphy107j** [comment]
Not sure on issues then no chat addons it just gets disabled every time when  I try to add chat window 4 with a reload ui/ load screen. I  use other addons but not sure if one is conflicting. I use adi bags atlasloot hedd dps  recount  kui nameplates  mik scrolling battle text world quest tracker stop resetting my camera  gathermate2 with data  handynotes garrison comander deadly boss mods. I hope we can solve this as it's annoying . I have also turned off my addons and still not working for me  I also made another post for the mythic plus timers not showing up the status of the run with how many mobs needed
